composer.json

```diff
@@ -28,7 +28,6 @@
     ],
     "require": {
         "php": "8.1.*||8.2.*||8.3.*||8.4.*",
-        "ext-zip": "*",
```

Comment:
This could be moved to the [`suggest` key](https://getcomposer.org/doc/04-schema.md#suggest) instead, although that would only make sense when installing PIE through composer, which maybe is a little odd order of things.